Mathlib/Order/Interval/Set/Disjoint.lean

```diff
@@ -43,8 +43,10 @@ theorem Iic_disjoint_Ioc (h : a ≤ b) : Disjoint (Iic a) (Ioc b c) :=
   (Iic_disjoint_Ioi h).mono le_rfl Ioc_subset_Ioi_self
 
 @[simp]
-theorem Ioc_disjoint_Ioc_same : Disjoint (Ioc a b) (Ioc b c) :=
-  (Iic_disjoint_Ioc le_rfl).mono Ioc_subset_Iic_self le_rfl
+theorem Ioc_disjoint_Ioc_of_le {d : α} (h : b ≤ c) : Disjoint (Ioc a b) (Ioc c d) :=
+  (Iic_disjoint_Ioc h).mono Ioc_subset_Iic_self le_rfl
+
+@[deprecated (since := "2025-03-04")] alias Ioc_disjoint_Ioc_same := Ioc_disjoint_Ioc_of_le
```

Comment:
Making it an `alias` might break people's code. Use the `deprecated` code action instead of the `deprecated alias ` code action